FearlessUtils/Classes/Crypto/SeedFactory.swift

```diff
@@ -6,6 +6,7 @@ public typealias SeedFactoryResult = (seed: Data, mnemonic: IRMnemonicProtocol)
 public protocol SeedFactoryProtocol {
     func createSeed(from password: String, strength: IRMnemonicStrength) throws -> SeedFactoryResult
     func deriveSeed(from mnemonicWords: String, password: String) throws -> SeedFactoryResult
+    func deriveNativeSeed(from mnemonicWords: String, password: String) throws -> SeedFactoryResult
```

Comment:
Le't move this implementation to a separate factory `BIP39SeedFactory` and also please implement `createSeed` function.

Reply:
Done